misc/python/materialize/mzcompose/__init__.py

```diff
@@ -88,6 +88,9 @@ def get_default_system_parameters(
         "compute_hydration_concurrency": "2",
         "compute_replica_expiration_offset": "3d",
         "compute_apply_column_demands": "true",
+        "compute_enable_peek_response_stash": "true",
+        # always use result stash, when eligible
+        "compute_peek_response_stash_threshold_bytes": "0",
```

Comment:
This seems dangerous because it completely disables testing of the inline results path in CI. Can we set this to a value that's small but not zero? Alternatively, @def- just added a way to randomize options in CI, maybe this would be a good use of that feature. https://github.com/MaterializeInc/materialize/pull/32679

Reply:
Yep, sorry, this was for getting maximal coverage for testing. I've been talking to Dennis about how we want to do this for real